"Added Dockerfile to dockerized the website"

RohanRusta21 opened the pull request:
HI @ghpranav @chandansgowda @Ramiz1336 @ShivamMenda @Sayed-Afnan-Khazi ,

I have created Dockerfile to Containerised the website which will help the website to get easily deploy on linux servers. I am actively participating in HacktoberFest 2023 :)

Please Review & Approve.

Thanks,
Rohan

RohanRusta21 committed:
"Added Dockerfile to dockerized the website"

chandansgowda commented on Dockerfile:
```diff
@@ -0,0 +1,7 @@
+FROM ubuntu
+RUN apt update -y
+LABEL maintainer="Rohan Rustagi"
```
We can't take your name as maintainer!

RohanRusta21 replied:
sure will remove it :)

chandansgowda commented on Dockerfile:
```diff
@@ -0,0 +1,7 @@
+FROM ubuntu
```
Cant we use alpine image?

RohanRusta21 replied:
yes we can, but the image is already very optimised and some dependencies doesn't work for alpine image. I don't have any issue using that, If you want I can use that.

chandansgowda commented on Dockerfile:
```diff
@@ -0,0 +1,7 @@
+LABEL maintainer="Rohan Rustagi"
+RUN apt install nginx -y
+COPY . /var/www/html
+EXPOSE 80
```
We will probably have to expose 443(https) too

RohanRusta21 replied:
sure will add it too.

RohanRusta21 committed:
"Modified Dockerfile as required"

RohanRusta21 commented:
Hi @chandansgowda @ghpranav @Ramiz1336 @ShivamMenda ,

I have modified the Dockerfile as per the above comments. Also documented the README.md file and added nginx.conf to securely expose port 443(https).

Please Review & Approve :)

Thanks,
Rohan

Sayed-Afnan-Khazi commented:
@chandansgowda, can we merge this?
